### ISSUE TITLE
badtouch 0.7.2 failed to set RLIMIT_NOFILE on mojave and catalina

### ISSUE THREAD

**Issue by chenrui333** [kpcyrd/badtouch#78]
```
$ /usr/local/Cellar/badtouch/0.7.2/bin/badtouch oneshot -vvx true.lua foo
[2021-01-14T05:00:37Z DEBUG badtouch::ulimit] soft_limit=256, hard_limit=9223372036854775807
[2021-01-14T05:00:37Z INFO  badtouch::ulimit] setting NOFILE limit to 9223372036854775807
Error: Failed to set RLIMIT_NOFILE

Caused by:
    Invalid argument (os error 22)
```

full log, https://github.com/Homebrew/homebrew-core/runs/1683397269
relates to https://github.com/Homebrew/homebrew-core/pull/68806

**Comment by carlocab** [comment]
Related: https://github.com/kpcyrd/badtouch/issues/76

**Comment by kpcyrd** [comment]
Sorry for the inconvenience, this has been fixed in 0.7.3.

Thanks for your work!
